java/ql/src/Security/CWE/CWE-094/InsecureBeanValidation.ql

```diff
@@ -38,5 +70,8 @@ class BeanValidationConfig extends TaintTracking::Configuration {
 }
 
 from BeanValidationConfig cfg, DataFlow::PathNode source, DataFlow::PathNode sink
-where cfg.hasFlowPath(source, sink)
-select sink, source, sink, "Custom constraint error message contains unsanitized user data"
+where
+  not exists(SetSafeMessageInterpolator ma) and
```

Comment:
Can you show me where in the code this gets set? Is there a way to determine whether a particular `setMessageInterpolator` call applies to the validation of a particular constraint?

Reply:
:+1: Will send you an example in Slack